SFM.py

```diff
@@ -184,15 +185,15 @@ def adj_list(self, matches=None):
                     self.adj[i][j] = 1
 
         return self.adj, pairs
-
     
-    def reconstruction(self):
-        if self.K is None:
-            raise AssertionError("Error: Camera matrix K not defined")
-        
-        # initial rec
-        # best_pair = sfm_helpers.Rec.best_pair(self.adj, self.good, self.kp, self.K)
-        
+    def get_camera_info(self):
+        for key in self.itoimg:
+            file = self.itoimg[key]
+            ext = os.path.splitext(file)[-1].lower()
```

Comment:
ion think u need this bc the itoimg only has image files

Reply:
need to get camera parameters which can error out depending on if there are text files in the source directory created from the output functions. The loop breaks once an image is found, which is usually in the 1st iter so it shouldn't be an issue. Separated in case we need to obtain params for a later step.

Reply:
itoimg has the filenames which is what we need

Reply:
yeah ik im saying keep `camera = pycolmap.infer_camera_from_image(f'{self.src}/{file}')` but drop the ext stuff

Reply:
its there to prevent errors, like reading another file which is not an image

Reply:
yeah but itoimg can only have image files tho